user/views.py

```diff
@@ -1,3 +1,444 @@
+                    description = data['description'],
+                )
+
+                return JsonResponse({'message' : 'SUCCESS'}, status = 200)
+
+        except KeyError as ex:
+            return JsonResponse({'message' : 'KEY_ERROR_' + ex.args[0]}, status = 400)
+        except Exception as ex:
+            return JsonResponse({'message' : 'ERROR_' + ex.args[0]}, status = 400)
+    
+    @access_decorator
+    def get(self, request): # 늘 사는 것 조회
+        try:
+            token = request.headers.get('Authorization', None)
+            payload = jwt.decode(token, SECRET, algorithm=ALGORITHM)
+            user = User.objects.filter(user_id = payload['user_id']).get().id
+
+            product_list = list(FrequentlyPurchasedProduct.objects.filter(user_id = user).values())
+
+            for item in product_list:
+                product = Product.objects.filter(id = item['product_id']).get()
+                item['name']  = product.name
+                item['price'] = product.price
+            
+            return JsonResponse({'message' : 'SUCCESS', 'product_list' : product_list}, status = 200)
+
+        except Exception as ex:
+            return JsonResponse({'message' : 'ERROR_' + ex.args[0]}, status = 400)
+
+    @access_decorator
+    def delete(self, request): # 늘 사는 것 목록 삭제
+        try:
+            data = json.loads(request.body)
+
+            item = FrequentlyPurchasedProduct.objects.filter(id = data['product_id']).get()
+            item.delete()
+
+            return JsonResponse({'message' : 'SUCCESS'}, status = 200)
+
+        except KeyError as ex:
+            return JsonResponse({'message' : 'KEY_ERROR_' + ex.args[0]}, status = 400)
+        except Exception as ex:
+            return JsonResponse({'message' : 'ERROR_' + ex.args[0]}, status = 400)
+
+class UserReview(View): # 유저의 상품 리뷰
+    @access_decorator
+    def post(self, request): # 유저의 상품 리뷰 등록
+        try: ############### 하나의 상품 하나의 유저 하나의 리뷰 예외처리 ############
+            data = json.loads(request.body)
+
+            Review.objects.create(
+                name        = data['name'],
+                help_count  = 0,
+                views_count = 0,
+                content     = data['content'],
+                user        = User(id = data['user_id']),
+                product     = Product(id = data['product_id']),
+                image_url   = data['image_url'],
+            )
+
+            return JsonResponse({'message' : 'SUCCESS'}, status = 200)
+
+        except KeyError as ex:
+            return JsonResponse({'message' : 'KEY_ERROR_' + ex.args[0]}, status = 400)
+        except Exception as ex:
+            return JsonResponse({'message' : 'ERROR_' + ex.args[0]}, status = 400)
+        
+class ProductReview(View):
+    def post(self, request): # 상품의 리뷰 상세보기 클릭 시 조회 수 증가
+        try:
+            data = json.loads(request.body)
+
+            review = Review.objects.filter(id = data['review_id']).get()
```

Comment:
```suggestion
            review = Review.objects.get(id = data['review_id'])
```

해당 리뷰가 존재하지 않을 때의 경우의 수도 에러로 잡아주셔야 할 것 같습니다.

Reply:
네 예외처리했습니다.